client/state/reader/follows/reducer.js

```diff
@@ -67,6 +69,44 @@ function updatePostSubscription( state, { payload, type } ) {
 			...follow,
 			delivery_methods: {
 				email: newFollowState,
+				notification: get( follow, [ 'delivery_methods', 'notification' ], {} ),
+			},
+		},
+	};
+}
+
+function updateNotificationSubscription( state, { payload, type } ) {
+	const follow = find( state, { blog_ID: +payload.blogId } );
+	if ( ! follow ) {
+		return state;
+	}
+
+	const currentFollowState = get( follow, [ 'delivery_methods', 'notification' ], {} );
+
+	const newProps = {};
+	switch ( type ) {
```

Comment:
I'm lost as to why this switch statement exists?
could it be replaced with:
```js
const newProps = {
  send_posts:  type !== READER_UNSUBSCRIBE_TO_NEW_POST_NOTIFICATIONS
};
```

Reply:
It's an artefact from `updateEmailSettings`. Removed now 👍